tideways/config.php-profiler.php

```diff
@@ -36,9 +36,36 @@
     ),
     // Other config
     'profiler.enable' => function() {
-        if ( isset( $_SERVER['REQUEST_URI'] ) && strpos( $_SERVER['REQUEST_URI'], 'vvv.test/' ) === 0 ) { //phpcs:ignore
-            return false;
+        $is_vvv_tideways = false;
+
+        if ( empty( $_SERVER['REMOTE_ADDR'] ) and !isset( $_SERVER['HTTP_USER_AGENT'] ) and count( $_SERVER['argv']) > 0 ) {
+            // CLI
+            if ( isset( $_SERVER['ENABLE_TIDEWAYS_CLI'] ) && $_SERVER['ENABLE_TIDEWAYS_CLI'] === '1' ) {
+                $is_vvv_tideways = true;
+            }
+        } else {
+            // Web requests:
+            if ( !isset( $_SERVER['HTTP_HOST'] ) ) {
+                return;
+            }
+
+            if ( strpos( $_SERVER['HTTP_HOST'], 'vvv.test' ) != false ) {
+                return;
+            }
+
+            if ( file_exists( '/srv/config/tideways.json' ) && in_array( $_SERVER['HTTP_HOST'], json_decode( file_get_contents( '/srv/config/tideways.json' ) ) ) ) {
```

Comment:
could this fail with PHP warnings/notices if tideways.json has invalid JSON? Maybe it should be split across more lines

Reply:
we generate the json with a PHP script using the same methods so I don't think that is something that can happen.
Anyway I added a try/catch for those cases

Reply:
true but people do weird things and might try to manually edit it